README.md

```diff
@@ -51,6 +51,12 @@ emitter.on('foo', onFoo)   // listen
 emitter.off('foo', onFoo)  // unlisten
 ```
 
+### Typescript
+```ts
+import * as mitt from 'mitt';
+let emitter: mitt.Emitter = new mitt();
```

Comment:
With the "new" keyword or without it, both variants don't work for me with TS. It either says "mitt is not a function" or "mitt is not a constructor" at runtime.